readthedocs_build/config/parser.py

```diff
@@ -0,0 +1,27 @@
+import yaml
+
+
+__all__ = ('parse', 'ParseError')
+
+
+class ParseError(Exception):
+    pass
+
+
+def parse(stream):
+    """
+    Take file-like object and return a list of project configurations.
+
+    The files need be valid YAML and only contain mappings as documents.
+    Everything else raises a ``ParseError``.
+    """
+    try:
+        configs = list(yaml.load_all(stream))
```

Comment:
Good catch. I've addressed this in the last commit: https://github.com/rtfd/readthedocs-build/commit/59c80b41d0b82d1da3b59ff9c3fb642b81b583e7